x-pack/plugins/actions/server/lib/action_executor.test.ts

```diff
@@ -511,6 +513,34 @@ test('logs a warning when alert executor returns invalid status', async () => {
   );
 });
 
+test('return executed action info', async () => {
+  setupActionExecutorMock();
+
+  await actionExecutor.logCancellation({
+    actionId: 'action1',
+    relatedSavedObjects: [],
+    request: {} as KibanaRequest,
+  });
+  expect(eventLogger.logEvent).toHaveBeenCalledTimes(2);
```

Comment:
I'm curious what the 2 calls are, but we're just checking the first.  Without seeing the whole file, perhaps just the way these work?